docs/index.md

```diff
@@ -7,19 +7,19 @@ Bing or other servers compliant to OGC standards like WFS, CSW, WMC, WMS, WMTS a
-**Application Context** where users can save, manage and share its own resources by also managing access permissions to other groups of users.
+[MapStore](https://mapstore.geo-solutions.it/mapstore/#/) resources are not only related to Maps but also [Dashboards](https://mapstore.readthedocs.io/en/latest/user-guide/exploring-dashboards/) and [Stories](https://mapstore.readthedocs.io/en/latest/user-guide/exploring-stories/); in [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) you can create your own innovative and fascinating 
+[Application Context](https://mapstore.readthedocs.io/en/latest/user-guide/managing-contexts/) where users can save, manage and share its own resources by also managing access permissions to other groups of users.
 
-[MapStore](https://mapstore.geo-solutions.it/mapstore/#/) is not only a product but also a WebGIS framework. As a standard geoportal product, it is a web-based product that allows to provide  
-a powerful and interactive geospatial WebGIS, it provides a direct and real-time access to geospatial data warehouses and it supports all available 
-standards formats for geospatial data. 
+[MapStore](https://mapstore.geo-solutions.it/mapstore/#/) is not only a product but also a WebGIS framework. As a standard geoportal product, it is a web-based product that allows to provide a powerful and interactive geospatial WebGIS, it provides a direct and real-time access to geospatial data warehouses and it supports all available standards formats for geospatial data. 
 [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) also provides advanced spatial analysis capabilities that can be used to build WebGIS solutions through a powerful, dynamic and open 
 geospatial application. Since [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) is also a framework, you can use it to build your own WebGIS applications by using its plugins and modules.
 
 Last but not the least, [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) is map agnostic and ensures the greatest flexibility: its abstraction tier allows to [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) to work with different web mapping libraries.
 The supported mapping engines currently supported by [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) are [OpenLayers](https://openlayers.org/) (used by default for desktops), [LeafletJS](https://leafletjs.com/) (used by default for mobile devices) and [Cesium 3D viewer](https://cesiumjs.org/).
 
-MapStore has been designed from the beginning to provide a coherent and comprehensive experience across different devices types.
+[MapStore](https://mapstore.geo-solutions.it/mapstore/#/) has been designed from the beginning to provide a coherent and comprehensive experience across different devices types.
```

Comment:
I would like to have this in the same page but under a title like "Supported Browsers" just before the "Quick Start" section.